Source/WebKit/UIProcess/API/Cocoa/_WKWebExtensionPermission.h

```diff
@@ -75,6 +75,10 @@ WK_EXTERN _WKWebExtensionPermission const _WKWebExtensionPermissionNativeMessagi
 WK_API_AVAILABLE(macos(WK_MAC_TBA), ios(WK_IOS_TBA), visionos(WK_XROS_TBA))
 WK_EXTERN _WKWebExtensionPermission const _WKWebExtensionPermissionScripting;
 
+/*! @abstract The `sidePanel` permission requests access to the `chrome.sidePanel` APIs. */
+WK_API_AVAILABLE(macos(WK_MAC_TBA), ios(WK_IOS_TBA))
+WK_EXTERN _WKWebExtensionPermission const _WKWebExtensionPermissionSidePanel;
```

Comment:
wouldn't we also need a permission for `sidebar_action`?

Reply:
I don't believe so -- the behavior in Firefox/Opera is that extensions are always allowed to use `sidebarAction` as long as their manifest specifies a `sidebar_action` key. Would I need to add anything to parse this? In my unit tests, it does seem to be visible in the manifest dictionary without any changes.

Reply:
i see. so there's no actual `sidebar_action` permission, just the entry in the manifest?

Reply:
Yep, exactly